### PR TITLE
script to show button text added

### PR DESCRIPTION
**Pull request by rafprzygoda**

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var classButton = document.getElementsByClassName("button");
+        console.log(classButton);
+        var i;
+        var text = "";
+        for (i = 0; i < classButton.length; i++) {
+            text += classButton[i].innerText + "<br>";      
+        }
+        document.getElementById("button_names").innerHTML = text;
```
Comment (matwjablonski) [inline js/script.js]: why you add <br> to innerText and then you set it to innerHTML? better if you add it to innerHTML in loop

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var classButton = document.getElementsByClassName("button");
+        console.log(classButton);
+        var i;
```
Comment (matwjablonski) [inline js/script.js]: this declaration move to loop for

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by rafprzygoda** [comment]
Fixed

File: js/script.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-        console.log(classButton);
-        var i;
-        var text = "";
-        for (i = 0; i < classButton.length; i++) {
-            text += classButton[i].innerText + "<br>";      
-        }
-        document.getElementById("button_names").innerHTML = text;
\ No newline at end of file
+var text = "";
+for (i = 0; i < classButton.length; i++) {
```
Comment (matwjablonski) [inline js/script.js]: `for (var i = 0; i < classButton.length; i++)` 

not below

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: js/script.js (hunk 3 above)
Comment (rafprzygoda) [reply js/script.js]: var text = ""; moved

Comment (rafprzygoda) [reply js/script.js]: I think now is ok :)

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**